src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -161,8 +161,8 @@
  * }
  * </pre>
  * <p>
- * To configure the check with policy {@code alone_or_singleline} for {@code if}
- * and <a href="apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">
+ * To configure the check with policy {@code alone_or_singleline} for {@code if} and
+ * <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF">
```

Comment:
can we avoid `https://checkstyle.org/` in url ?

Reply:
Please fix this meanwhile

Reply:
@romani Actually we can't avoid `https://checkstyle.org/` in the URL because when we avoid it, `https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/checks/blocks/` gets added before `apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_DEF` which makes up an incorrect URL and does not exist.

I also tried one more thing but it doesn't work so we may have to just keep it the same as in the commit.